src/Components/Timer/Timer.js

```diff
@@ -33,6 +38,13 @@ const CountdownTimer = () => {
                 lastUnit='m'
                 startImmediately={false}
                 timeToUpdate={10}
+                formatValue={(value) => `${(value < 10? `0${value}` : value)}`}
```

Comment:
Assuming I'm reading this correctly, we might want to remove this line! I think this will add a `0` placeholder back to the beginning of the minutes.

No need to do this before we merge!

Reply:
Oh yeah! good point!!

Reply:
Great catch, just removed & pushed up that change!